invenio_sip2/models.py

```diff
@@ -462,6 +462,32 @@ def has_magnetic_media(self):
         return self.get('magnetic_media')
 
 
+class SelfcheckFeePaid(dict):
+    """Class representing fee paid handler response."""
+
+    def __init__(self, accepted=False, **kwargs):
+        """Constructor.
+
+        :param accepted: payment operation is accepted or not
+            should be set to True if the ACS accept the payment.
+            should be set to 0 if the ACS did not accept the payment.
+        :param kwargs: optional fields
+        """
+        # required properties
+        self['payment_accepted'] = accepted
+        self['screen_messages'] = []
+
+        # optional properties
+        for key, value in kwargs.items():
+            if value:
+                self[key] = value
+
+    @property
+    def is_accepted(self):
+        """Shortcut for payment_accepted operation."""
+        return self.get('payment_accepted')
```

Comment:
should return a default boolean value ?

Reply:
this attribute is set to false in constructor if the is no value.
I made the change anyway.